src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/shipping.html.twig

```diff
@@ -65,6 +65,38 @@
         </span>
         </div>
       </div>
+      <div class="form-group row {% if not recycledPackagingEnabled %}d-none{% endif %}">
+        <div class="col-3">
+        <span class="float-right">
+          {{ 'Recycled packaging'|trans({}, 'Admin.Orderscustomers.Feature') }}
+        </span>
+        </div>
+        <div class="col-4">
+          <span class="ps-switch">
```

Comment:
There is already a ps-switch widget available :thinking:  (bootstrap_4_layout.html.twig)

Reply:
Yes it is the Form Theme 🤔 

I am not aware it can be reused inside Twig template, outside of a `form_widget` statement. Can you point me to the right documentation about that ? I tried three Symfony documentation pages but all of them mention template inheritance or Form Theme usage, neither fit this usecase.

Reply:
If it's an element in the form, it should be in the form widget no? :thinking:

Reply:
This part of the page is not a form 😞 . The "Create an Order" page is a hybrid page where the for is slowly being built in ajax following user inputs. First you select a customer, which loads the "add product" or "reuse a cart/order" blocks, then you create a cart which unlocks 'shipping', 'voucher' blocks ... until finally 'summary' block is displayed.

So the part of the code in this PR is HTML parts being hidden at first, then hydrated with cart data by JS and loaded when user reaches this part of the workflow.

Now that we look at it 🤔 we could have done better ( @zuk3975 was talking about it yesterday) , we have been influenced by how the legacy page was built and we somehow followed the same structure.

To make it short: this part is not a regular form, which is why it's not rendered by a `form_widget`.

Reply:
In this PR I followed how the initial block was built (see https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/shipping.html.twig#L59)